packages/cli/package.json

```diff
@@ -12,17 +12,17 @@
     "@pattern-lab/core": "^5.14.3",
     "@pattern-lab/live-server": "^5.14.3",
     "@pattern-lab/starterkit-mustache-base": "3.0.3",
-    "archiver": "2.1.1",
+    "archiver": "5.3.0",
     "chalk": "4.1.0",
     "commander": "2.15.1",
-    "deepmerge": "^2.1.1",
-    "execa": "4.0.3",
-    "fs-extra": "6.0.1",
+    "deepmerge": "^4.2.2",
+    "execa": "5.0.0",
+    "fs-extra": "10.0.0",
     "glob": "7.1.6",
-    "has-yarn": "1.0.0",
-    "inquirer": "5.1.0",
+    "has-yarn": "2.1.0",
+    "inquirer": "8.0.0",
     "lodash": "4.17.21",
-    "ora": "2.1.0",
+    "ora": "5.4.0",
```

Comment:
This is a huge version skip. Aren't there no side effects or breaking changes in the CLI?
This also applies to inquirer with 3 major version increases.

Reply:
I've checked all of those packages release by release within their respective changelogs. On the things that needed an update, I've listed those within the related "Updated and migrated some code" section of the description.

 I would like to suggest to collect the breaking changes for each of those breaking releases and add them in the above listing as well for a better overview.